README.md

```diff
@@ -54,11 +54,26 @@ git clone https://github.com/indieagi/jupyterhub.git
 cd jupyterhub/
 ```
 
+### Set-up Environment Variables
+If you are setting up the *Dev environment*, run:
+```
+cd ./install-scripts
+
+chmod +x install-docker.sh start-docker-on-boot.sh set-dev-env-variables.sh
```

Comment:
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Delete lines 60, 62, 67, 69 and replace with just two lines instead of four:
```
cd install-scripts
chmod +x *.sh
```
[rest of the instructions]